### ISSUE TITLE
Mockup for Surf Paddle

### ISSUE THREAD

**Issue by julang0707** [CheeseFry/SurfingProject#3]
![surf-paddle](https://cloud.githubusercontent.com/assets/13006914/8730796/c5240fd2-2bb8-11e5-89f6-c23a9546cd1b.png)

@CheeseFry

**Comment by CheeseFry** [comment]
So I need a site that looks like this?
Are there rules about the functionality?

**Comment by julang0707** [comment]
I put the details is Issue #5.  There is an easy and a hard mode.  Hard mode is the responsive version.

**Comment by awilli1186** [comment]
Yes take this mock up with the provided assets and tools to recreate the above image into a responsive website. Clickable links,
